tests/e2e/package.json

```diff
@@ -27,7 +27,7 @@
     "@types/shelljs": "^0.8.11",
     "axios": "^0.25.0",
     "chai": "^4.3.4",
-    "chromedriver": "^111.0.0",
+    "chromedriver": "^114.0.2",
```

Comment:
@nallikaea I don't seem to be able to find this change in the dockerfile as well. Please make sure the versions are consistent and the image builds successfully.

Reply:
In Dockerfile we still have `FROM selenium/standalone-chrome:107.0` is this line take any action? =)
https://github.com/eclipse/che/blob/3dd17c762c1f17ca85616418d2d886ac42d55b87/tests/e2e/build/dockerfiles/Dockerfile#L1

Reply:
updated

Reply:
Perfect, thank you!